helm/prometheus-meta-operator/templates/prometheus-rules/prometheus.yml

```diff
@@ -1,22 +1,19 @@
-[[ if eq .ClusterType "control_plane" ]]
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   name: prometheus
-  namespace: [[ .Namespace ]]
+  namespace: {{ include "resource.default.namespace" . }}
   labels:
-    cluster_id: [[ .ClusterID ]]
-    app.kubernetes.io/name: "prometheus"
-    app.kubernetes.io/managed-by: [[ .ManagedBy ]]
-    app.kubernetes.io/instance: [[ .ClusterID ]]
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "control_plane"
 spec:
   groups:
   - interval: 0s
     name: prometheus
     rules:
     - alert: PrometheusCantCommunicateWithTenantAPI
       annotations:
-        description: Prometheus can't communicate with tenant cluster {{ $labels.cluster_id }} Kubernetes API.
```

Comment:
Why did you remove this cluster id here?

Reply:
Apart from this, this looks great if tested

Reply:
Because this is actually automagically added by alertmanager and opsgenie, and shows up in the alert title.

Reply:
Cool :)